### PR TITLE
Added some debug logs in mimecast collector

### PR DESCRIPTION
**Pull request by sameerbag**
### Problem Description
No debug logs in mimecast collector

### Solution Description
Added debug logs in mimecast collector

### CODE REVIEW

File: collectors/mimecast/utils.js [hunk 1]
```diff
@@ -222,13 +226,16 @@ function generateHeaders(authDetails, uri) {
...
-    return {
+    let returnObj = {
         "Authorization": `MC ${authDetails.accessKey}:${signature}`,
         "x-mc-app-id": authDetails.appId,
         "x-mc-date": hdrDate,
         "x-mc-req-id": requestId,
         "Content-Type": 'application/json'
     };
+
+    AlLogger.debug('MIME000012 url header: ', returnObj);
```
Comment (kkuzmin) [inline collectors/mimecast/utils.js:237]: please exclude `Authorization ` header from debug log

**Review by kkuzmin — CHANGES_REQUESTED** [review]
please bump package.json version

File: collectors/mimecast/utils.js (hunk 1 above)
Comment (sameerbag) [reply collectors/mimecast/utils.js]: Done, @kkuzmin please review it again.

**Comment by sameerbag** [comment]
> 

Done

**Review by kkuzmin — APPROVED** [review]

**Pull request merged**